src/renderer/components/theme-settings/theme-settings.js

```diff
@@ -117,8 +119,10 @@ export default defineComponent({
         // Third group
         this.$t('Settings.Theme Settings.Base Theme.Catppuccin Mocha'),
         this.$t('Settings.Theme Settings.Base Theme.Dracula'),
+        this.$t('Settings.Theme Settings.Base Theme.Gruvbox Dark'),
+        this.$t('Settings.Theme Settings.Base Theme.Gruvbox Light'),
         this.$t('Settings.Theme Settings.Base Theme.Solarized Dark'),
-        this.$t('Settings.Theme Settings.Base Theme.Solarized Light')
+        this.$t('Settings.Theme Settings.Base Theme.Solarized Light'),
```

Comment:
Added by accident?

```suggestion
        this.$t('Settings.Theme Settings.Base Theme.Solarized Light')
```

Reply:
Yes. But then this line would need a change as well, right?

https://github.com/FreeTubeApp/FreeTube/pull/5887/files/8761459021d563296ffa737bdc0f3f56c443baa5#diff-bca2ce5e9aaaac8354224d694b202bf079b2c78e2ecf6405ebb56ace7608b5abR107